doc/api/process.md

```diff
@@ -1426,13 +1426,25 @@ added: v0.1.16
 * {string}
 
 The `process.platform` property returns a string identifying the operating
-system platform on which the Node.js process is running. For instance
-`'darwin'`, `'freebsd'`, `'linux'`, `'sunos'` or `'win32'`
+system platform on which the Node.js process is running.
+
+Currently possible values are:
+
+* `'aix'`
+* `'darwin'`
+* `'freebsd'`
+* `'linux'`
+* `'openbsd'`
+* `'sunos'`
+* `'win32'`
 
 ```js
 console.log(`This platform is ${process.platform}`);
 ```
 
+The value `'android'` may also be returned if the Node.js is built on the
+Android operating system. However, Android support in Node.js is experimental.
```

Comment:
@marswong , in reply to https://github.com/nodejs/node/pull/18057#discussion_r160843070

Yes, I was replying to @Fishrock123 's suggestion that we link to the supported platforms table. 

Maybe something like this:

>Android support in Node.js [is experimental](https://github.com/nodejs/node/blob/master/BUILDING.md#supported-platforms-1)